sources/presentation/Stride.Core.Presentation.Avalonia/MarkupExtensions/DoubleExtension.cs

```diff
@@ -16,9 +18,9 @@ public DoubleExtension()
         Value = 0.0;
     }
 
-    public DoubleExtension(object value)
+    public DoubleExtension(string value)
```

Comment:
I'd prefer to keep an `object` argument so that it eventually works for any type that can be converted.